darts/models/forecasting/forecasting_model.py

```diff
@@ -12,14 +12,17 @@
 to obtain forecasts for a desired number of time stamps into the future.
 """
 import copy
+import datetime
 import inspect
+import os
 import time
 from abc import ABC, ABCMeta, abstractmethod
 from collections import OrderedDict
 from itertools import product
 from random import sample
-from typing import Any, Callable, Dict, List, Optional, Sequence, Tuple, Union
+from typing import Any, BinaryIO, Callable, Dict, List, Optional, Sequence, Tuple, Union
 
+import joblib
```

Comment:
Do joblib or BinaryIO require new dependencies?

Reply:
no, joblib is a dependency of sklearn and BinaryIO is part of the typing module.